Tests/PayPalMessagesTests/PayPalMessageModalTests.swift

```diff
@@ -101,6 +101,7 @@ class PayPalMessageModalTests: XCTestCase {
         let closeButtonAvailableHeight = 70
         let closeButtonColor = UIColor(hexString: "#001435")
         let closeButtonColorType = "light"
+        let closeButtonAlternativeText = "PayPal Modal Close"
```

Comment:
noting that `closeButtonAlternativeText` text here is inconsistent with text in `Sources/PayPalMessages/Config/PayPalMessageModalConfig.swift`